Formula/qscintilla2.rb

```diff
@@ -12,14 +12,17 @@ class Qscintilla2 < Formula
   end
 
   option "without-plugin", "Skip building the Qt Designer plugin"
+  option "without-python", "Skip building the python bindings"
 
   depends_on :python => :recommended
```

Comment:
It's a little counterintuitive, but you can go ahead and remove this line; the presence of `option "without-python"` will do the right thing